TFT/src/User/API/RRFParseACK.cpp

```diff
@@ -154,11 +150,14 @@ void ParseACKJsonParser::value(const char *value)
 
 void rrfParseACK(const char *data)
 {
+  static JsonStreamingParser parser;
+  static ParseACKJsonParser handler;
```

Comment:
I see you defined multiple instances for parser and handler. Isn't it possible to share them across more files?

Reply:
There's really only once instead of each handler in global memory (static) one ParseACKJsonParser and one RRFM20Parser. So there's nothing to share there. At some point, I will probably change all usages of RRM20Parser to be a streaming variant, and then it will be shared instance. (stack vs. heap)

The problem with sharing JsonStreamingParser is that it has to maintain current document state (start/end, etc.). Thinking about it, it doesn't seem like it would be reentrant as there is only 1 active document at any given time, and it's either a something being parsed by parseACK or being passed onto cmd_rev_buf/stream_handler. I'll probably make that change shortly.